docs/storing-querying/storage-options/index.md

```diff
@@ -40,12 +46,41 @@ Data warehouses and data lakes are primary destinations for Snowplow data. For o
 <AzureExperimental/>
 
 | Destination | Type | Loader application | Status |
-| --- | --- | --- |
+| --- | --- | --- | --- |
 | Snowflake | Micro-batching | [Snowplow RDB Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/index.md) | Early release |
+| Databricks | Micro-batching<br/>_(via a [data lake](#data-lake-loaders))_ | LAKELOADERLINK | Early release |
+| Synapse Analytics | Micro-batching<br/>_(via a [data lake](#data-lake-loaders))_ | LAKELOADERLINK | Early release |
 
   </TabItem>
 </Tabs>
 
-### Data lake loaders
+## Data lake loaders
+
+All lake loaders are micro-batching.
+
+<Tabs groupId="cloud" queryString>
+  <TabItem value="aws" label="AWS" default>
+
+| Lake | Format | Compatibility | Loader application | Status |
+| --- | --- | --- | --- | --- |
+| S3 | TSV | Amazon Athena | [S3 Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/s3-loader/index.md) | Production-ready |
```

Comment:
IMO we should not be encouraging users to query TSV with Athena.  It's a not production-appropriate way to query data.

At this point, the only reason to use the S3 loader is as a pre-cursor to the RDB batch transformer.  I do not see any other reason to use it.

**correction** -- actually, we should advise it a solution for querying failed events (because we don't have any better solution).  But not enriched events.